packages/action-bar/src/ActionBar.ts

```diff
@@ -23,13 +24,16 @@ import '@spectrum-web-components/button/sp-close-button.js';
 import '@spectrum-web-components/field-label/sp-field-label.js';
 import actionBarStyles from './action-bar.css.js';
 import { ifDefined } from '@spectrum-web-components/base/src/directives.js';
+import { FocusVisiblePolyfillMixin } from "@spectrum-web-components/shared/src/focus-visible.js";
 export const actionBarVariants = ['sticky', 'fixed'];
 
 /**
  * @element sp-action-bar
  * @slot - Content to display with the Action Bar
  */
-export class ActionBar extends SpectrumElement {
+export class ActionBar extends SizedMixin(
+    FocusVisiblePolyfillMixin(SpectrumElement)
```

Comment:
Seems reasonable for the close button. Would be great if UXP prioritized adding this, as I'm sure there are lots of little places where UI like this falls through the cracks and keeping the polyfill code for something that has been fixed in browser for so long is questionable on our end.